packages/design-tokens/README.md

```diff
@@ -1,5 +1,3 @@
-<div align="center">
```

Comment:
Why did you remove this (and not the closing tag)?

I prefer the title centered.

Reply:
Oops this was a mistake, thanks 🙏 I was gonna verify everything on the deploy preview anyways